src/Vocento/Composer/Installers/Installer.php

```diff
@@ -79,6 +87,18 @@ private function getIO()
      */
     public function supports($packageType)
     {
-        return 'vocento-magento-core' === $packageType;
+        $packageType = strtolower($packageType);
+
+        return array_key_exists($packageType, self::SUPPORTED_TYPES);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getPackageInstaller(PackageInterface $package)
```

Comment:
This could be move to a new class like `PackageInstallerFactory` or something like that and inject the supported types.
Making this change, this class would be more testable and reusable.